common/src/python/centers/center_group.py

```diff
@@ -522,6 +549,71 @@ def __add_user_roles_to_project(self, *, user: User, project_id: str,
 
         return project.add_user_roles(user=user, roles=roles)
 
+    def __add_user_to_redcap_project(
```

Comment:
Let's hide the details of redcap and parameter store within the center group object.

A reasonable strategy is to create a `REDCapProjectRepository`  `get(redcap_pid) -> REDCapProject` and add a field for an object within the center group class along side the FlywheelProxy reference.